core/templates/pages/practice-session-page/practice-session-page.constants.ts

```diff
@@ -20,9 +20,11 @@ export class PracticeSessionPageConstants {
   public static TOTAL_QUESTIONS = 20;
 
   public static PRACTICE_SESSIONS_DATA_URL =
-    '/practice_session/data/<topic_name>';
+    '/practice_session/data/<topic_name>?' +
+    'selected_subtopics=<subtopic_titles_list>';
```

Comment:
It's not a list, it's a json string, so that should be reflected in the param name.

Reply:
Renamed it